python/cuspatial/cuspatial/core/binpreds/contains.py

```diff
@@ -12,7 +15,69 @@
 from cuspatial.utils.column_utils import normalize_point_columns
 
 
-def contains_properly(
+def contains_properly_quadtree(
+    test_points,
+    polygons,
+):
```

Comment:
Suggest renaming to `quedtree_contains_properly` and `pairwise_conains_properly` for consistency.

```suggestion
def contains_properly_quadtree(
    test_points: GeoSeries,
    polygons: GeoSeries,
):
```